app/views/content/blog/getting-ready-to-apply.md

```diff
@@ -73,7 +73,9 @@ If you’re an undergraduate student interested in teaching physics, maths, comp
 
 ## 6. Start lining up your references
 
-You’ll need a couple of referees to provide references for your application. Now is a good time to start approaching people who might be able to provide these for you. Getting references in place is one of the most common causes of delay in the application process, so any preparation you can do now could speed things up considerably when you make your application.
+You’ll need a couple of references for your application. Your references will confirm to providers that you're a good fit for teaching.
```

Comment:
As I understand it, you don't need references "for your application"? Does this need wording slightly differently?